beacon_node/network/src/sync/range_sync/batch.rs

```diff
@@ -525,3 +530,29 @@ impl<E: EthSpec> std::fmt::Debug for BatchState<E> {
         }
     }
 }
+
+impl<E: EthSpec> BatchState<E> {
+    /// Creates a unicode visualisation for the batch state to display in logs for quicker and
+    /// easier recognition
+    ///
+    /// The current icons are:
+    /// - Empty/Uninitialized: E
+    /// - Downloading: D
+    /// - Awaiting Download: d
+    /// - Awaiting Validation: V
+    /// - Failed: F
+    /// - AwaitingProcessing: p
+    /// - Processing(_) => "P",
+    /// - Poisoned => "X",
+    fn visualize(&self) -> char {
+        match self {
+            BatchState::Downloading(_, _) => 'D',
+            BatchState::Processing(_) => 'P',
+            BatchState::AwaitingValidation(_) => 'V',
```

Comment:
I think the proposed convention is to use lower case for all `Awaiting-` states. Or are you using `V` here because we don't have a conflict for 'v'?